src/models/maps/bravia/defaultKeys.ts

```diff
@@ -0,0 +1,351 @@
+		name: 'textbox',
+		tap_action: { action: 'textbox' },
+		icon: 'mdi:text-box',
+	},
+	{
+		type: 'button',
+		name: 'search',
+		tap_action: { action: 'search' },
+		icon: 'mdi:google-assistant',
+	},
+	{
+		type: 'button',
+		name: 'enter',
+		tap_action: { action: 'key', key: 'Confirm' },
+		icon: 'mdi:magnify',
+	},
+	{
+		type: 'button',
+		name: 'channel_up',
+		tap_action: { action: 'key', key: 'ChannelUp' },
+		icon: 'mdi:arrow-up-circle',
+	},
+	{
+		type: 'button',
+		name: 'channel_down',
+		tap_action: { action: 'key', key: 'ChannelDown' },
+		icon: 'mdi:arrow-down-circle',
+	},
+	{
+		type: 'button',
+		name: 'red',
+		tap_action: { action: 'key', key: 'Red' },
+		icon: 'mdi:alpha-r-box',
+	},
+	{
+		type: 'button',
+		name: 'green',
+		tap_action: { action: 'key', key: 'Green' },
+		icon: 'mdi:alpha-g-box',
+	},
+	{
+		type: 'button',
+		name: 'yellow',
+		tap_action: { action: 'key', key: 'Yellow' },
+		icon: 'mdi:alpha-y-box',
+	},
+	{
+		type: 'button',
+		name: 'blue',
+		tap_action: { action: 'key', key: 'Blue' },
+		icon: 'mdi:alpha-b-box',
+	},
+  {
+		type: 'button',
+		name: 'n0',
+		tap_action: { action: 'key', key: 'Num0' },
+		icon: 'mdi:numeric-0',
+	},
+	{
+		type: 'button',
+		name: 'n1',
+		tap_action: { action: 'key', key: 'Num1' },
+		icon: 'mdi:numeric-1',
+	},
+	{
+		type: 'button',
+		name: 'n2',
+		tap_action: { action: 'key', key: 'Num2' },
+		icon: 'mdi:numeric-2',
+	},
+	{
+		type: 'button',
+		name: 'n3',
+		tap_action: { action: 'key', key: 'Num3' },
+		icon: 'mdi:numeric-3',
+	},
+	{
+		type: 'button',
+		name: 'n4',
+		tap_action: { action: 'key', key: 'Num4' },
+		icon: 'mdi:numeric-4',
+	},
+	{
+		type: 'button',
+		name: 'n5',
+		tap_action: { action: 'key', key: 'Num5' },
+		icon: 'mdi:numeric-5',
+	},
+	{
+		type: 'button',
+		name: 'n6',
+		tap_action: { action: 'key', key: 'Num6' },
+		icon: 'mdi:numeric-6',
+	},
+	{
+		type: 'button',
+		name: 'n7',
+		tap_action: { action: 'key', key: 'Num7' },
+		icon: 'mdi:numeric-7',
+	},
+	{
+		type: 'button',
+		name: 'n8',
+		tap_action: { action: 'key', key: 'Num8' },
+		icon: 'mdi:numeric-8',
+	},
+	{
+		type: 'button',
+		name: 'n9',
+		tap_action: { action: 'key', key: 'Num9' },
+		icon: 'mdi:numeric-9',
+	},
+	{
+		type: 'button',
+		name: 'numpad',
+		icon: 'mdi:dialpad',
+	},
+	{
+		type: 'button',
+		name: 'captions',
+		tap_action: { action: 'key', key: 'SubTitle' },
+		icon: 'mdi:closed-caption',
+	},
+	{
+		type: 'button',
+		name: 'explorer',
+		tap_action: { action: 'key', key: 'ApplicationLauncher' },
+		icon: 'mdi:folder-multiple',
+	},
+	{
+		type: 'button',
+		name: 'teletext',
+		tap_action: { action: 'key', key: 'Teletext' },
+		icon: 'mdi:card-text',
+	},
+	{
+		type: 'button',
+		name: 'tv',
+		tap_action: { action: 'key', key: 'Tv' },
+		icon: 'mdi:television-box',
+	},
+	{
+		type: 'button',
+		name: 'audio_track',
+		tap_action: { action: 'key', key: 'MediaAudioTrack' },
+		icon: 'mdi:waveform',
+	},
+	{
+		type: 'button',
```

Comment:
While this does have a button, it's really a source and should be in the other list.